lib/js/stanford_publication.behavior.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Behavior Example that works with Webpack.
+ *
+ * @see: https://www.npmjs.com/package/drupal-behaviors-loader
+ *
+ * Webpack wraps everything in enclosures and hides the global variables from
+ * scripts so special handling is needed.
+ */
+
+export default {
+
+  // Attach Drupal Behavior.
+  attach: function (context, settings) {
+    (function ($) {
+
+      // only open and close filter menu when on responsive size.
+      const mediaQuery = window.matchMedia('(max-width: 991px)')
+
+      function handleTabletChange(e) {
+        // Check if the media query is true
+        if (e.matches) {
+
+          // Remove the h2 and replace with button.
+          $("#system-menu-blockstanford-publication-topics-menu")
+            .contents()
+            .unwrap()
+            .wrap('<button class="publication-topics__collapsable-menu"></button>');
```

Comment:
you don't want to unwrap it if you're in desktop. only unwrap if its in mobile AND wrapped in the button tag.

Reply:
It doesn't unwrap it because it is in the `if` on line 21. 
<img width="915" alt="Screen Shot 2021-03-22 at 3 46 57 PM" src="https://user-images.githubusercontent.com/1187335/112067809-f0817980-8b25-11eb-9425-31c52b9c35f4.png">

Reply:
`unwrap` removes the immediate parent. so if you load the page fresh on a desktop and the markup is something like
```
<div class="wrapper">
  <div class="menu">
    <div id="system-menu-blockstanford-publication-topics-menu">
    </div>
  </div>
</div>
```
the result would be 
```
<div class="wrapper">
  <button class="publication-topics__collapsable-menu">
    <div id="system-menu-blockstanford-publication-topics-menu">
    </div>
  </button>
</div>
```

Notice the missing `menu` class wrapper.

Reply:
You can also provide an argument to the `unwrap` function so you could do `unwrap('.publication-topics__collapsable-menu')`
 >>> selector
Type: String
A selector to check the parent element against. If an element's parent does not match the selector, the element won't be unwrapped.

Reply:
Ooops. Fixed it with the argument.